package.json

```diff
@@ -19,9 +21,12 @@
     "markdownz": "~4.0.2",
     "modal-form": "~2.3.0",
     "moment": "~2.9.0",
-    "panoptes-client": "~2.0.0",
+    "panoptes-client": "git://github.com/CKrawczyk/panoptes-javascript-client.git#add-stats-server",
```

Comment:
Why is the stats server defined in panoptes-client if it's not used there? Should an interface with the stats server interface be part of panoptes-client?